### PR TITLE
feat: add static access lists for the recipe system

### PR DESCRIPTION
**Pull request by kjaredic**

### CODE REVIEW

File: src/Action.js [hunk 1]
```diff
@@ -132,6 +134,14 @@ class Action {
   async getEthValue() {
     return '0';
   }
+
+  /**
+   * Access list for single action
+   * @returns {Promise<Array<*>>}
```
Comment (yippee-ki-yay) [inline src/Action.js:140]: Does not return promise

File: src/Recipe.js [hunk 2]
```diff
@@ -99,6 +99,43 @@ class Recipe {
       .reduce((acc, val) => acc.add(new BN(val)), new BN(0))
       .toString();
   }
+
+  /**
+   * Generates an access list for the recipe
+   * @returns {Promise<Array<*>>}
```
Comment (yippee-ki-yay) [inline src/Recipe.js:105]: same same here

File: src/Action.js (hunk 1 above)
Comment (kjaredic) [reply src/Action.js]: true

File: src/Recipe.js (hunk 2 above)
Comment (kjaredic) [reply src/Recipe.js]: also true

**Pull request merged**